40.meta-mender/docs.md

```diff
@@ -7,6 +7,98 @@ shortcode-core:
 github: false
 ---
 
+## meta-mender zeus-v2021.07
+
+_Released 07.23.2021_
+
+### Statistics
+
+A total of 1314 lines added, 298 removed (delta 1016)
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 12 (40.0%) |
+| Lluis Campos | 11 (36.7%) |
+| Ole Petter Orhagen | 2 (6.7%) |
+| Fabio Tranchitella | 2 (6.7%) |
+| Pierre-Jean Texier | 1 (3.3%) |
+| Corey Cothrum | 1 (3.3%) |
+| Alan Martinovic | 1 (3.3%) |
+
+| Developers with the most changed lines | |
+|---|---|
+| Kristian Amlie | 1009 (76.6%) |
+| Lluis Campos | 238 (18.1%) |
+| Fabio Tranchitella | 55 (4.2%) |
+| Ole Petter Orhagen | 12 (0.9%) |
+| Pierre-Jean Texier | 1 (0.1%) |
+| Corey Cothrum | 1 (0.1%) |
+| Alan Martinovic | 1 (0.1%) |
+
+| Developers with the most signoffs (total 6) | |
+|---|---|
+| Lluis Campos | 6 (100.0%) |
+
+| Top changeset contributors by employer | |
+|---|---|
+| Northern.tech | 28 (93.3%) |
+| texier.pj2@gmail.com | 1 (3.3%) |
+| contact@coreycothrum.com | 1 (3.3%) |
+
+| Top lines changed by employer | |
+|---|---|
+| Northern.tech | 1315 (99.8%) |
+| texier.pj2@gmail.com | 1 (0.1%) |
+| contact@coreycothrum.com | 1 (0.1%) |
+
+| Employers with the most signoffs (total 6) | |
+|---|---|
+| Northern.tech | 6 (100.0%) |
```

Comment:
why only 1 of 6 we show?

Reply:
That means that 6 signoffs (100%) are coming from NT